apps/docs/src/common/pages/accessibility/audio-and-video.tsx

```diff
@@ -0,0 +1,224 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A, NavigationMenu } from '@not-govuk/components';
+
+const Page: FC<PageProps> = ({ location }) => (
+  <div className="govuk-grid-row">
+    <Helmet>
+      <title>Accessibility - Home Office Design System</title>
+      <meta name="og:article:section" content="Accessibility" />
+    </Helmet>
+    <div className="govuk-grid-column-one-quarter">
+      <NavigationMenu items={[
+        {
+          href: '/accessibility/audio-and-video',
+          text: 'Audio and video'
+        }
+      ]} />
+    </div>
+    <div className="govuk-grid-column-three-quarters">
+      <h1>
+        <span className="caption">Accessibility</span>
+        Audio and video
+      </h1>
+      <p>It's important to consider how people with different access needs will use media content.</p>
+
+
+      <table class="govuk-table">
```

Comment:
In the `.tsx` files, we need to change `class=` to `className=`. So this line and the ones below it need to change.

Alternatively you could import the `Table` component and use that: https://not-gov.uk/components?name=Table

At the top of the file you would update:
```ts
import { A, NavigationMenu } from '@not-govuk/components';
```
to:
```ts
import { A, NavigationMenu, Table } from '@not-govuk/components';
```

Then you could adapt the code snippets on the website I linked to.

Reply:
I've changed everything from 'class' to 'className'.